client/blocks/login/two-factor-authentication/push-notification-illustration.jsx

```diff
@@ -35,27 +38,45 @@ function DeviceSvg() {
 
 function NotificationSvg() {
 	return (
-		<svg className="two-factor-authentication__illustration-notification">
-			<g fill="none" fillRule="evenodd">
+		<svg
+			xmlns="http://www.w3.org/2000/svg"
+			fill="none"
+			viewBox="0 0 310 80"
+			className="two-factor-authentication__illustration-notification jetpack"
```

Comment:
I noticed that we include the `jetpack` CSS class. However, I couldn't find what styles are added to the element when I inspected the HTML, so I'm wondering if we need it, wdyt?

Reply:
You're right, it's no longer needed, thanks for catching! Removed that in https://github.com/Automattic/wp-calypso/pull/70832/commits/53043bb141909b01b394fdca626e3a1dce9569ff.